cmd/legacy/heartbeat/params.go

```diff
@@ -305,19 +295,19 @@ func readExtraHeartbeats() ([]heartbeat.Heartbeat, error) {
 func parseExtraHeartbeat(data string) ([]heartbeat.Heartbeat, error) {
 	var incoming []struct {
 		Category          heartbeat.Category `json:"category"`
-		CursorPosition    *int               `json:"cursorpos"`
+		CursorPosition    interface{}        `json:"cursorpos"`
 		Entity            string             `json:"entity"`
 		EntityType        string             `json:"entity_type"`
 		Type              string             `json:"type"`
-		IsWrite           *bool              `json:"is_write"`
```

Comment:
What values will be accepted as true/false?

Reply:
Any bool or string. `true`, `false`, `"true"`and `"false"`.